share/functions/funced.fish

```diff
@@ -102,14 +102,28 @@ function funced --description 'Edit function definition'
     else
         echo $init >$tmpname
     end
+
     # Repeatedly edit until it either parses successfully, or the user cancels
     # If the editor command itself fails, we assume the user cancelled or the file
     # could not be edited, and we do not try again
     while true
+        if which md5sum > /dev/null
+            set checksum (md5sum $tmpname)
```

Comment:
Note that it appears BSDs (and macOS) don't have an "md5sum" program, but an `md5` one. So this function would be disabled on those systems. Which is a good way of _failing_, but it would be nicer if it didn't.

I'll leave it to those using those systems to come up with the correct incantation to use here.